statsd/src/main/java/com/avast/metrics/statsd/StatsDMetricsMonitor.java

```diff
@@ -36,7 +36,7 @@ public StatsDMetricsMonitor(String host, int port, boolean autoRegisterMetrics,
         this.naming = naming;
         this.gaugeSendPeriod = gaugeSendPeriod;
         this.scheduler = scheduler;
-        client = createStatsDClient(host, port, prefix);
+        client = createStatsDClient(host, port, sanitizeNames(prefix));
```

Comment:
I believe `.` is a valid character for the prefix.

Reply:
good point, will modify